flutter_frontend_server/BUILD.gn

```diff
@@ -80,27 +79,3 @@ application_snapshot("frontend_server") {
 
   inputs = frontend_server_files
 }
-
-# For flutter/flutter#36738 we make the source files available so that
-# we can generate a local frontend_server snapshot in the tools cache.
-action("package_incremental_compiler") {
-  script = "//flutter/flutter_frontend_server/package_incremental.py"
-
-  inputs = frontend_server_files
-
-  outputs = [
-    "$root_gen_dir/dart-pkg/flutter_frontend_server/pubspec.yaml",
```

Comment:
I think you probably still want to generate these? Not totally sure but they will likely be read in by the script.

Reply:
This isn't actually used right now, so its really just one more thing that could break with null safety :)